submissions/damilolaSalami/html-form/contactUs.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
```

Comment:
kindly work on your existing contact form page. Make changes there. do not create a new file

Reply:
kindly delete this file